### PR TITLE
Wallbox documentation (related to HA PR 48082)

### PR DESCRIPTION
**Pull request by hesselonline**
related to HA PR 48082 (https://github.com/home-assistant/core/pull/48082), documentation for new wallbox sensor component.

## Proposed change
Add new documentation for wallbox integration (HA PR 48082_

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: related to HA PR 48082 (https://github.com/home-assistant/core/pull/48082)
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2353
- This PR fixes or closes issue: 

## Checklist

- [ X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wallbox.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+This `wallbox` sensor platform pulls data from https://my.wallbox.com for your wallbox charging station.
+
+## Configuration
+
+To enable this sensor in your installation, go to Configuration and Integrations and add a MyWallbox integration. You will need to fill in your wallbox:
+- serial number
+-  your username (email)
+-  password
```
Comment (klaasnicolaas) [inline source/_integrations/wallbox.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/wallbox.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Wallbox
+description: Instructions on how to integrate sensors Wallbox EV chargers to Home Assistant
+ha_category:
+  - Car
+ha_release: 2021.4
+ha_iot_class: Cloud Polling
+ha_domain: wallbox
+ha_platforms:
+  - sensor
```
Comment (klaasnicolaas) [inline source/_integrations/wallbox.markdown]: Add here `ha_config_flow: true`

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/wallbox.markdown (hunk 1 above)
Comment (hesselonline) [reply source/_integrations/wallbox.markdown]: @klaasnicolaas, thnx for the review. I included your proposals

File: source/_integrations/wallbox.markdown [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ha_config_flow: true
+---
+
+This `wallbox` sensor platform pulls data from https://my.wallbox.com for your wallbox charging station.
```
Comment (frenck) [inline source/_integrations/wallbox.markdown]:
```suggestion
The Wallbiox integration pulls data from the [MyWallbox Portal](https://my.wallbox.com) for your Wallbox charging station.
```

File: source/_integrations/wallbox.markdown [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The integration adds the following sensors:
```
Comment (frenck) [inline source/_integrations/wallbox.markdown:20]:
```suggestion
The integration adds the following sensors:

```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/wallbox.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: Wallbox
+description: Instructions on how to integrate sensors Wallbox EV chargers to Home Assistant
+ha_category:
+  - Car
+ha_release: 2021.4
```
Comment (frenck) [inline source/_integrations/wallbox.markdown]:
```suggestion
ha_release: 2021.6
```

**Review by frenck — APPROVED** [review]

**Pull request merged**